src/main/java/com/minecolonies/apiimp/initializer/EntityInitializer.java

```diff
@@ -111,6 +118,35 @@ public static void setupEntities()
                                       .size(0.98F, 0.7F)
                                       .build(Constants.MOD_ID + ":mcminecart")
                                       .setRegistryName("mcminecart");
+
+        ModEntities.MUMMY = EntityType.Builder.create(EntityMummy::new, EntityClassification.MONSTER)
+                               .setTrackingRange(ENTITY_TRACKING_RANGE)
+                               .setUpdateInterval(ENTITY_UPDATE_FREQUENCY)
+                               .size((float) CITIZEN_WIDTH, (float) CITIZEN_HEIGHT)
+                               .build(Constants.MOD_ID + ":mummy")
+                               .setRegistryName("mummy");
+
+        ModEntities.ARCHERMUMMY = EntityType.Builder.create(EntityArcherMummy::new, EntityClassification.MONSTER)
+                                     .setTrackingRange(ENTITY_TRACKING_RANGE)
+                                     .setUpdateInterval(ENTITY_UPDATE_FREQUENCY)
+                                     .size((float) CITIZEN_WIDTH, (float) CITIZEN_HEIGHT)
+                                     .build(Constants.MOD_ID + ":archermummy")
+                                     .setRegistryName("archermummy");
+
+        ModEntities.PHARAO = EntityType.Builder.create(EntityPharao::new, EntityClassification.MONSTER)
```

Comment:
Did you try a different size for the boss? might look nice

Reply:
No, didn't try yet =D